Doesn't compile on Ubuntu 20.04

I spun up a fresh Ubuntu 20.04 VM, and built ImageMagick manually:

```
sudo apt install build-essential pkgconf clang lldb
sudo ln -s /usr/bin/llvm-config-10 /usr/local/bin/llvm-config
wget http://www.imagemagick.org/download/ImageMagick.tar.gz
tar -xzvf ImageMagick.tar.gz
cd ImageMagick-7.0.10-33/
./configure
make
sudo make install
```

I then tried to build `magick-rust`:

```
git clone https://github.com/nlfiedler/magick-rust.git
cd magick-rust/
curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
cargo build --release
```

And got this output:

```
$ cargo build --release
   Compiling magick_rust v0.14.0 (/home/ubuntu/magick-rust)
error: failed to run custom build command for `magick_rust v0.14.0 (/home/ubuntu/magick-rust)`

Caused by:
  process didn't exit successfully: `/home/ubuntu/magick-rust/target/release/build/magick_rust-95e13b5e81b2af38/build-script-build` (exit code: 101)
  --- stdout
  cargo:rerun-if-env-changed=IMAGE_MAGICK_LIB_DIRS
  cargo:rerun-if-env-changed=IMAGE_MAGICK_DIR
  cargo:rerun-if-env-changed=MAGICKWAND_NO_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=MAGICKWAND_STATIC
  cargo:rerun-if-env-changed=MAGICKWAND_DYNAMIC
  cargo:rerun-if-env-changed=PKG_CONFIG_ALL_STATIC
  cargo:rerun-if-env-changed=PKG_CONFIG_ALL_DYNAMIC
  ... snip lots of similar lines ...
  --- stderr
  /usr/local/include/ImageMagick-7/MagickCore/magick-config.h:63:3: warning: "you should set MAGICKCORE_QUANTUM_DEPTH to sensible default set it to configure time default" [-W#warnings]
  /usr/local/include/ImageMagick-7/MagickCore/magick-config.h:64:3: warning: "this is an obsolete behavior please fix your makefile" [-W#warnings]
  /usr/local/include/ImageMagick-7/MagickCore/magick-config.h:74:5: error: invalid token at start of a preprocessor expression
  /usr/local/include/ImageMagick-7/MagickCore/magick-config.h:86:3: warning: "you should set MAGICKCORE_HDRI_ENABLE to sensible default set it to configure time default" [-W#warnings]
  /usr/local/include/ImageMagick-7/MagickCore/magick-config.h:87:3: warning: "this is an obsolete behavior please fix yours makefile" [-W#warnings]
  /usr/local/include/ImageMagick-7/MagickCore/magick-config.h:113:5: error: invalid token at start of a preprocessor expression
  /usr/local/include/ImageMagick-7/MagickCore/magick-type.h:64:6: error: invalid token at start of a preprocessor expression
  /usr/local/include/ImageMagick-7/MagickCore/magick-type.h:77:8: error: invalid token at start of a preprocessor expression
  /usr/local/include/ImageMagick-7/MagickCore/magick-type.h:90:8: error: invalid token at start of a preprocessor expression
  /usr/local/include/ImageMagick-7/MagickCore/magick-type.h:103:8: error: invalid token at start of a preprocessor expression
  /usr/local/include/ImageMagick-7/MagickCore/magick-type.h:112:2: error: "MAGICKCORE_QUANTUM_DEPTH must be one of 8, 16, 32, or 64"
  /usr/local/include/ImageMagick-7/MagickCore/pixel.h:230:18: error: unknown type name 'Quantum'
  /usr/local/include/ImageMagick-7/MagickCore/pixel.h:234:47: error: unknown type name 'Quantum'
  /usr/local/include/ImageMagick-7/MagickCore/pixel.h:235:11: error: unknown type name 'Quantum'
  /usr/local/include/ImageMagick-7/MagickCore/pixel.h:244:11: error: unknown type name 'Quantum'
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:66:27: error: unknown type name 'Quantum'
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:69:52: error: expected ';' after top level declarator
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:69:53: warning: declaration does not declare anything [-Wmissing-declarations]
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:77:5: error: unknown type name 'Quantum'
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:79:33: error: unknown type name 'Quantum'
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:81:5: error: unknown type name 'Quantum'
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:93:21: error: unknown type name 'Quantum'
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:96:47: error: expected ';' after top level declarator
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:96:63: error: expected identifier or '('
  fatal error: too many errors emitted, stopping now [-ferror-limit=]
  /usr/local/include/ImageMagick-7/MagickCore/magick-config.h:63:3: warning: "you should set MAGICKCORE_QUANTUM_DEPTH to sensible default set it to configure time default" [-W#warnings], err
: false
  /usr/local/include/ImageMagick-7/MagickCore/magick-config.h:64:3: warning: "this is an obsolete behavior please fix your makefile" [-W#warnings], err: false
  /usr/local/include/ImageMagick-7/MagickCore/magick-config.h:74:5: error: invalid token at start of a preprocessor expression, err: true
  /usr/local/include/ImageMagick-7/MagickCore/magick-config.h:86:3: warning: "you should set MAGICKCORE_HDRI_ENABLE to sensible default set it to configure time default" [-W#warnings], err:
false
  /usr/local/include/ImageMagick-7/MagickCore/magick-config.h:87:3: warning: "this is an obsolete behavior please fix yours makefile" [-W#warnings], err: false
  /usr/local/include/ImageMagick-7/MagickCore/magick-config.h:113:5: error: invalid token at start of a preprocessor expression, err: true
  /usr/local/include/ImageMagick-7/MagickCore/magick-type.h:64:6: error: invalid token at start of a preprocessor expression, err: true
  /usr/local/include/ImageMagick-7/MagickCore/magick-type.h:77:8: error: invalid token at start of a preprocessor expression, err: true
  /usr/local/include/ImageMagick-7/MagickCore/magick-type.h:90:8: error: invalid token at start of a preprocessor expression, err: true
  /usr/local/include/ImageMagick-7/MagickCore/magick-type.h:103:8: error: invalid token at start of a preprocessor expression, err: true
  /usr/local/include/ImageMagick-7/MagickCore/magick-type.h:112:2: error: "MAGICKCORE_QUANTUM_DEPTH must be one of 8, 16, 32, or 64", err: true
  /usr/local/include/ImageMagick-7/MagickCore/pixel.h:230:18: error: unknown type name 'Quantum', err: true
  /usr/local/include/ImageMagick-7/MagickCore/pixel.h:234:47: error: unknown type name 'Quantum', err: true
  /usr/local/include/ImageMagick-7/MagickCore/pixel.h:235:11: error: unknown type name 'Quantum', err: true
  /usr/local/include/ImageMagick-7/MagickCore/pixel.h:244:11: error: unknown type name 'Quantum', err: true
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:66:27: error: unknown type name 'Quantum', err: true
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:69:52: error: expected ';' after top level declarator, err: true
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:69:53: warning: declaration does not declare anything [-Wmissing-declarations], err: false
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:77:5: error: unknown type name 'Quantum', err: true
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:79:33: error: unknown type name 'Quantum', err: true
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:81:5: error: unknown type name 'Quantum', err: true
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:93:21: error: unknown type name 'Quantum', err: true
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:96:47: error: expected ';' after top level declarator, err: true
  /usr/local/include/ImageMagick-7/MagickCore/cache-view.h:96:63: error: expected identifier or '(', err: true
  fatal error: too many errors emitted, stopping now [-ferror-limit=], err: true
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ()', build.rs:144:43
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

If I edit `/usr/local/include/ImageMagick-7/MagickCore/magick-config.h` to include this line:

    #define MAGICKCORE_QUANTUM_DEPTH 8

Then it compilation works fine. I'm not sure how to best handle that without manual editing, however.

--- a/build.rs
+++ b/build.rs
@@ -26,9 +26,7 @@ use std::process::Command;
 const MIN_VERSION: &'static str = "7.0";
 const MAX_VERSION: &'static str = "7.1";
 
-static HEADER: &'static str = r#"#define MAGICKCORE_QUANTUM_DEPTH 8
-#include <MagickWand/MagickWand.h>
-"#;
+static HEADER: &'static str = "#include <MagickWand/MagickWand.h>\n";
 
 fn main() {
     if cfg!(target_os = "freebsd") {
@@ -42,6 +40,15 @@ fn main() {
         env_var_set_default("IMAGE_MAGICK_LIB_DIRS", "/usr/local/lib");
         env_var_set_default("IMAGE_MAGICK_LIBS", "MagickWand-7");
     }
+
+    let cppflags = Command::new("MagickCore-config")
+        .arg("--cppflags")
+        .output()
+        .unwrap()
+        .stdout;
+    let cppflags = String::from_utf8(cppflags).unwrap();
+
+    env_var_set_default("BINDGEN_EXTRA_CLANG_ARGS", &cppflags);
 
     let lib_dirs = find_image_magick_lib_dirs();
     for d in &lib_dirs {

--- a/build.rs
+++ b/build.rs
@@ -26,7 +26,9 @@ use std::process::Command;
 const MIN_VERSION: &'static str = "7.0";
 const MAX_VERSION: &'static str = "7.1";
 
-static HEADER: &'static str = "#include <MagickWand/MagickWand.h>\n";
+static HEADER: &'static str = r#"#define MAGICKCORE_QUANTUM_DEPTH 8
+#include <MagickWand/MagickWand.h>
+"#;
 
 fn main() {
     if cfg!(target_os = "freebsd") {

